src/integration/utils/getPageTypeFromTestPath.js

```diff
@@ -0,0 +1,6 @@
+module.exports = testPath => {
+  const [pageType] = testPath.match(/(?<=\/integration\/pages\/).+?(?=\/)/g) ||
+    testPath.match(/(?<=\/__GENERATED_TEST_FILES__\/.+?\/).+?(?=\/)/g) || [''];
```

Comment:
Do we still have `__GENERATED_TEST_FILES__` in this PR?

Reply:
Nope. I fixed this